libraries/botbuilder-dialogs-adaptive/src/converters/multiLanguageRecognizerConverter.ts

```diff
@@ -6,21 +6,28 @@
  * Licensed under the MIT License.
  */
 
-import { Converter, ResourceExplorer } from 'botbuilder-dialogs-declarative';
+import { Converter } from 'botbuilder-dialogs';
+import { ResourceExplorer } from 'botbuilder-dialogs-declarative';
 import { Recognizer } from '../recognizers';
 import { RecognizerConverter } from './recognizerConverter';
 
-export class MultiLanguageRecognizerConverter implements Converter {
+type Input = Record<string, string>;
+type Output = Record<string, Recognizer>;
+
+export class MultiLanguageRecognizerConverter implements Converter<Input, Output> {
     private _recognizerConverter: RecognizerConverter;
 
     public constructor(resouceExplorer: ResourceExplorer) {
         this._recognizerConverter = new RecognizerConverter(resouceExplorer);
     }
 
-    public convert(value: object): { [key: string]: Recognizer } {
+    public convert(value: Input | Output): Output {
         const recognizers = {};
```

Comment:
Use `reduce` when constructing an object like this.

```typescript
return Object.entries(value).reduce((recognizers, [key, value]) => {
  if (!Object.prototype...) {
    return recognizers;
  }
  return { ...recognizers, [key]: this._recognizerConverter.convert(value) };
}, {});
```

I actually think using `Object.entries` handles the `hasOwnProperty` filtering implicitly.